pkg/operator/controllers/guardrails/guardrails_config.go

```diff
@@ -41,11 +41,11 @@ const (
 
 	defaultManagerRequestsCPU = "100m"
 	defaultManagerLimitCPU    = "1000m"
-	defaultManagerRequestsMem = "256Mi"
+	defaultManagerRequestsMem = "512Mi"
```

Comment:
out of curiosity - is this following a recommendation per the version bump, or something unrelated?

Reply:
Hi @yjst2012 I am also curious about this. Because it will have impact on a scheduler and the masters are of a fixed size.

Reply:
yep, this change is made by gatekeeper. well basically all stuff here is porting from upstream.